crates/wasm/src/circuit.rs

```diff
@@ -2,29 +2,6 @@ use acvm::acir::circuit::Circuit;
 use gloo_utils::format::JsValueSerdeExt;
 use wasm_bindgen::prelude::*;
 
-// Deserializes bytes into ACIR structure
-#[deprecated(
-    note = "we have moved away from this serialization strategy. Call `acir_read_bytes` instead"
-)]
-#[allow(deprecated)]
-#[wasm_bindgen]
-pub fn acir_from_bytes(bytes: Vec<u8>) -> JsValue {
-    console_error_panic_hook::set_once();
-    let circuit = Circuit::from_bytes(&bytes);
-    <JsValue as JsValueSerdeExt>::from_serde(&circuit).unwrap()
-}
-
-#[deprecated(
-    note = "we have moved away from this serialization strategy. Call `acir_write_bytes` instead"
-)]
-#[allow(deprecated)]
-#[wasm_bindgen]
-pub fn acir_to_bytes(acir: JsValue) -> Vec<u8> {
-    console_error_panic_hook::set_once();
-    let circuit: Circuit = JsValueSerdeExt::into_serde(&acir).unwrap();
-    circuit.to_bytes()
-}
```

Comment:
@TomAFrench @kobyhallx Are we fine to remove these deprecated functions with the acvm 0.9.0 upgrade?

Reply:
Yes, we're not using this anymore and they got removed as they've been deprecated since 0.2.1: https://github.com/noir-lang/acvm/pull/177